src/setup.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, session):
             getConfigListEntry(_("User"), config_root.user),
             getConfigListEntry(_("Password"), config_root.password),
             getConfigListEntry(_("Sitemap"), config_root.sitemap),
+            getConfigListEntry(_("Data refresh *100ms"), config_root.refresh),
```

Comment:
*100ms is confusing for the users. They don't know what "10" value means, it is not obvious that "10" means one second. It should be either seconds or milliseconds.

Reply:
Repertuaru to 1s, done